data/json/recipes/recipe_food.json

```diff
@@ -3913,9 +3914,9 @@
       [ "cabbage", 1 ],
       [ "irradiated_cabbage", 1 ],
       [ "sauerkraut", 2 ],
+      [ "can_corn", 1 ],
```

Comment:
@mugling Ya I should have written why. So they would display in recipes in the same pattern.